modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -241,6 +244,40 @@ class Configuration(object):
         # Disable client side validation
         self.client_side_validation = True
 
+    @classmethod
+    def copy(cls, source):
```

Comment:
How about changing this to:
`__deepcopy__(self, memodict={})`
That way you control how everything can be copied. For the logger only you can then make a new logger with the same properties as the old one. All other properties can be deep copied. You can use dir(self) or `self.__dict__` to iterate over all of self's properties. 

If you do it this way, then in new_instance you can return copy deepcopy(cls._default)

Reply:
Yeah, I'd totally forgotten about this possibility. Thanks.